cfme/fixtures/parallelizer/__init__.py

```diff
@@ -581,14 +580,13 @@ def provs_of_tests(test_group):
                 prov = provs[0]
                 # Already too many slaves with provider
                 app = slave.appliance
-                self.print_message(
-                    'cleansing appliance', slave, purple=True)
+                self.print_message('removing providers from appliance', slave, purple=True)
                 try:
                     app.delete_all_providers()
                 except Exception as e:
-                    self.print_message(
-                        'cloud not cleanse', slave, red=True)
-                    self.print_message('error:', e, red=True)
```

Comment:
Passing `e` here was triggering an exception in print_message, as we were calling `decode` on it if it wasn't a string.

There's an assumption in print_message where its calling decode on anything that isn't a string, assuming its binary string.

I'm not addressing the behavior in print_message at the moment, as it will soon be impacted by py3 conversion.